packages/frontend-2/components/common/Card.vue

```diff
@@ -21,20 +21,25 @@
       v-if="buttons"
       class="flex flex-col flex-wrap md:flex-row gap-y-2 md:gap-x-2 gap-y-0 mt-3"
     >
-      <FormButton
+      <div
         v-for="(button, index) in buttons"
         :key="button.id || index"
-        v-bind="button.props || {}"
-        :disabled="button.props?.disabled || button.disabled"
-        :submit="button.props?.submit || button.submit"
-        target="_blank"
-        external
-        size="sm"
-        color="outline"
-        @click="($event) => button.onClick?.($event)"
+        v-tippy="button.disabledMessage"
```

Comment:
I see you have mastered the ways of v-tippy

Reply:
praise be